src/video_core/texture_cache/texture_cache.h

```diff
@@ -1,12 +1,15 @@
-// SPDX-FileCopyrightText: Copyright 2019 yuzu Emulator Project
-// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-FileCopyrightText: 2021 yuzu emulator team
+// (https://github.com/skyline-emu/)
+// SPDX-License-Identifier: GPL-3.0-or-later Licensed under GPLv3
+// or any later version Refer to the license.txt file included.
```

Comment:
Here you mention "yuzu emulator team" (instead of "yuzu Emulator Project" btw) but then link to Skyline... I guess you meant to add copyright notices for both

```suggestion
// SPDX-FileCopyrightText: 2021 yuzu Emulator Project
// SPDX-FileCopyrightText: 2021 Skyline Team and Contributors <https://github.com/skyline-emu>
// SPDX-License-Identifier: GPL-3.0-or-later
```